publish/publish-type.proj

```diff
@@ -9,7 +9,7 @@
     <BuildDependsOn>
       ValidateProperties;
       DownloadFilesFromContainer;
-      SignSymbolPackages
+      PublishSymbols
```

Comment:
Does the new target actually handle signing the symbols correctly now or is that not needed any longer?